rotkehlchen/data_migrations/migrations/migration_6.py

```diff
@@ -0,0 +1,87 @@
+import json
+import logging
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from rotkehlchen.constants.misc import ONE
+from rotkehlchen.fval import FVal
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import SupportedBlockchain
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.drivers.gevent import DBCursor
+    from rotkehlchen.rotkehlchen import Rotkehlchen
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+DEAD_NODES_RPCS = [
+    'https://api.mycryptoapi.com/eth',  # mycrypto
+    'https://mainnet-nethermind.blockscout.com/',  # blockscout
+    'https://mainnet.eth.cloud.ava.do/',  # avado
+    'https://web3.1inch.exchange',  # 1inch
+    'https://names.mewapi.io/rpc/eth',  # myetherwallet
+]
+
+
+def update_nodes_in_database(write_cursor: 'DBCursor', rotki: 'Rotkehlchen') -> None:
+    """
+    Updates the ethereum nodes in the database. If nodes in the db are exactly the same as the
+    defaults (i.e. the user has not customized them) then replace them with the new defaults.
+    Else just delete the dead nodes and reweigh.
+    """
+    dir_path = Path(__file__).resolve().parent.parent.parent
+    with open(dir_path / 'data' / 'nodes_as_of_1-26-1.json', 'r') as f:
+        old_nodes_info = json.loads(f.read())
+    with open(dir_path / 'data' / 'nodes.json', 'r') as f:
+        new_nodes_info = json.loads(f.read())
+
+    old_nodes = {
+        (node['name'], node['endpoint'], node['owned'], node['weight'])
+        for node in old_nodes_info
+    }
+
+    nodes_in_db = set(write_cursor.execute(
+        'SELECT name, endpoint, owned, weight FROM web3_nodes WHERE blockchain=?',
+        (SupportedBlockchain.ETHEREUM.value,),
+    ))
+
+    if nodes_in_db == old_nodes:
+        # If nodes in the db are exactly same as old defaults, replace them with new defaults
+        log.info('Completely replacing ethereum rpc nodes with new defaults')
+        write_cursor.execute(
+            'DELETE FROM web3_nodes WHERE blockchain=?',
+            (SupportedBlockchain.ETHEREUM.value,),
+        )
+        new_nodes_tuples = [
+            (node['name'], node['endpoint'], node['owned'], node['active'], str(FVal(node['weight'])), node['blockchain'])  # noqa: E501
+            for node in new_nodes_info
+        ]
+        write_cursor.executemany(
+            'INSERT INTO web3_nodes(name, endpoint, owned, active, weight, blockchain) VALUES (?, ?, ?, ?, ?, ?)',  # noqa: E501
+            new_nodes_tuples,
+        )
+    else:
+        # Else just delete dead nodes
+        log.info('Deleting dead ethereum rpc nodes but keeping the other ones')
+        write_cursor.execute(
+            f'DELETE FROM web3_nodes WHERE endpoint IN ({",".join("?"*len(DEAD_NODES_RPCS))}) AND blockchain=?',  # noqa: E501
+            DEAD_NODES_RPCS + [SupportedBlockchain.ETHEREUM.value],
```

Comment:
no need to do list adddition here. This is better and gives it as a tuple and is easier to match to the actual arguments
```suggestion
            (*DEAD_NODES_RPCS, SupportedBlockchain.ETHEREUM.value),
```